docs/docs/sample-scripts/entra/find-obsolete-m365-groups/index.mdx

```diff
@@ -0,0 +1,288 @@
+
+      $Group.SharePointStatus = @{
+        Reason = $reason
+      }
+      $Group.TestStatus = "🟡 Warning"
+      $Group.Reasons += $reason
+
+      try {
+        $Global:ObsoleteGroups.Add($Group.Reference.id, $Group)
+      }
+      catch {}  
+    }  
+  }
+
+  function Test-ConversationActivity {
+    [CmdletBinding()]
+    param (
+      [Parameter(Mandatory = $true)] [GroupInfo] $Group
+    )
+
+    $WarningDate = (Get-Date).AddDays(-365)
+      
+    $a = m365 entra m365group conversation list --groupId $Group.Reference.id | ConvertFrom-Json | Where-Object {
+      [datetime]$_.lastDeliveredDateTime -lt $WarningDate
+    }
+    if (!$a -or $a.Length -eq 0) { return }
+
+    Write-Host "  → potentially obsolete ($($a.Length) conversation item$($a.Length -gt 1 ? 's' : '') created more than 1 year ago)" -ForegroundColor Yellow
+    $reason = "$($a.Length) conversation item$($a.Length -gt 1 ? 's' : '') created more than 1 year ago)"
+
+    $Group.MailboxStatus = @{
+      OutdatedConversations = $a | Sort-Object -Property lastDeliveredDateTime
+      Reason                = $reason
+    }
+    $Group.TestStatus = "🟡 Warning"
+    $Group.Reasons += $reason
+
+    try {
+      $Global:ObsoleteGroups.Add($Group.Reference.id, $Group)
+    }
+    catch { }
+  }
+
+  function New-Report {
+    [CmdletBinding()]
+    param (
+      [Parameter(Mandatory = $true)] [GroupInfo] $Group
+    )
+    
+    $exportObject = [ordered] @{
+      "Group Name"              = $Group.Reference.displayName
+      "Managed by"              = $Group.Membership.Owners ? $Group.Membership.Owners.displayName -join ", " : "n/a"
+      Owners                    = $Group.Membership.Owners.Count
+      Members                   = $Group.Membership.Members.Count
+      Guests                    = $Group.Membership.Guests.Count
+      "Group Status"            = $Group.Membership.Status ?? "Normal"
+      Description               = $Group.Reference.description
+      "Conversation Status"     = $Group.MailboxStatus.Reason ?? "Normal"
+      "Number of Conversations" = $Group.MailboxStatus.OutdatedConversations ? $Group.MailboxStatus.OutdatedConversations.Length : "n/a"
+      "Last Conversation"       = $Group.MailboxStatus.OutdatedConversations ? $Group.MailboxStatus.OutdatedConversations[0].lastDeliveredDateTime : ""
+      "Team enabled"            = $Group.Reference.resourceProvisioningOptions -contains 'Team' ? "True" : "False"
+      "SPO Status"              = $Group.SharePointStatus.Reason ?? "Normal"
+      "SPO Activity"            = $Group.SharePointStatus ? "Low / No document library usage" : "Document library in use"
+      "Number of warnings"      = $Group.Reasons.Count
+      Status                    = $Group.TestStatus
+    }
+
+    $exportObject | Export-Csv -Path $Script:ReportPath -Append -NoTypeInformation
+  }
+
+  # START the report generation
+  Start-Routine #-KeepConnectionsAlive -KeepOutputPath
```

Comment:
Seems like I'm unable to get the script working. It keeps crashing while checking all groups:

![image](https://github.com/user-attachments/assets/2f1d4d15-6eb8-43d6-9a31-30dee9a156b3)

Reply:
I will dig into this within the next few days, @milanholemans – I'm already a bit tired... 😃

Reply:
Can you provide some more details on this error, @milanholemans?

Reply:
I haven't really dug deeper into it. You're not experiencing this error?

Reply:
No, no issues so far... 🤷‍♂️

Reply:
I'll try to look into it ASAP.

Reply:
Thank you very much!